docker/ray-ml/Dockerfile

```diff
@@ -8,6 +8,7 @@ COPY requirements_dl.txt ./
 COPY requirements_ml_docker.txt ./
 COPY requirements_rllib.txt ./
 COPY requirements_tune.txt ./requirements_tune.txt
+COPY requirements_train.txt ./
```

Comment:
Keep this in alphabetical order?

Reply:
They are already not in alphabetical order...will leave this for a future PR

Reply:
Actually `ml_docker` has to come at the end

Reply:
Ok will make the change